src/openrct2/Game.cpp

```diff
@@ -588,6 +589,20 @@ void game_load_init()
     gGameSpeed = 1;
 }
 
+void game_load_scripts()
+{
+#ifdef __ENABLE_SCRIPTING__
+    GetContext()->GetScriptEngine().LoadPlugins();
+#endif
+}
+
+void game_finish()
+{
+#ifdef __ENABLE_SCRIPTING__
+    GetContext()->GetScriptEngine().UnloadPlugins();
+#endif
+}
+
```

Comment:
Hmm, wouldn't the `ifdef` outside of the functions look clearer?

Reply:
I generally do it this way to reduce #if defs in the headers and callers as well. Less guarding overall.

Reply:
Ok, I get it and can live with that.